adafruit_as726x.py

```diff
@@ -527,20 +536,20 @@ class AS726x_UART(AS726x):
 
     """
 
-    def __init__(self, uart):
+    def __init__(self, uart: busio.UART):
         self._uart = uart
         self._uart.baudrate = 115200
         super().__init__()
 
-    def read_channel(self, channel):
+    def read_channel(self, channel: int) -> int:
         """Read an individual sensor channel"""
         return self._virtual_read(channel)
 
-    def read_calibrated_value(self, channel):
+    def read_calibrated_value(self, channel: int) -> float:
         """Read a calibrated sensor channel"""
         return self._virtual_read(channel)
 
-    def _uart_xfer(self, cmd):
+    def _uart_xfer(self, cmd: int) -> int:
```

Comment:
Since it can potentially return `None`, the return here should use `Optional`, see below for followup.

Reply:
Also, `rstrip()` is used with strings (at least that's where I've used it), should the return type actually be `Optional[str]`?

Reply:
Also, it seems `cmd` should also be `str` if its using the `encode()` method and based on how its used elsewhere